open-craft/edx-platform
# Taxonomy import templates [FC-0036]

This PR adds a Studio endpoint for downloading the CSV and JSON templates added to oel_tagging by https://github.com/openedx/openedx-learning/pull/89

Closes https://github.com/openedx/modular-learning/issues/137

Private-ref: [FAL-3520](https://tasks.opencraft.com/browse/FAL-3520)

Depends on https://github.com/openedx/edx-platform/pull/33438

**Testing instructions**

Using this branch in your devstack's `cms-shell`:
1. Update requirements: `paver install_prereqs`
1. Update migrations: `./manage.py lms migrate`
2. Download the template files:
    http://localhost:18010/api/content_tagging/v1/taxonomies/import/template.json
    http://localhost:18010/api/content_tagging/v1/taxonomies/import/template.json
3. Check that the files download as expected.
